.github/workflows/CI.yml

```diff
@@ -40,6 +46,9 @@ jobs:
       - name: Build and test (OpenSSL)
         run: |
           dbus-run-session -- cargo test --no-default-features --features async-std --features openssl_crypto
+      - name: Build and test keyring-util
```

Comment:
we don't have tests for keyring-util, so just drop this

Reply:
Renamed to `Build` and used `cargo build` instead.